.pylintrc

```diff
@@ -12,8 +14,24 @@ max-branches=16
 
 [FORMAT]
 max-line-length = 99
-enable=no-absolute-import,old-division
-disable=cyclic-import,duplicate-code,fixme,import-outside-toplevel,locally-disabled,locally-enabled,missing-docstring,no-else-break,no-else-continue,no-else-raise,no-else-return,raise-missing-from,redefined-variable-type,stop-iteration-return,super-with-arguments,useless-object-inheritance
+enable=no-absolute-import,old-division,using-f-string-in-unsupported-version
```

Comment:
It is unnecessary to explicitly enable `using-f-string-in-unsupported-version` since we set `py-version` in `[MASTER]` section.